azure/blob-storage/main.tf

```diff
@@ -55,7 +55,7 @@ resource "azurerm_storage_account" "main" {
     }
   }
 
-  tags = local.default_tags
+  tags = coalesce(var.tags, local.default_tags)
```

Comment:
Just an idea, WDYT if we use `merge` to keep the default_tags if it is not override?

Reply:
I'm conflicted. What if users want to use **exactly** their custom tags, and won't expect any other tags to be added?

Reply:
Good point.